docs/diagrams/versions.svg

```diff
@@ -0,0 +1,44 @@
+link tfmodule to tfprovider--><path codeLine="29" d="M238.5268,52.0251 C257.2954,69.6207 283.2699,93.9718 302.1265,111.6498 " fill="none" id="tfmodule-tfprovider" style="stroke:#312D2A;stroke-width:1.0;"/><!--MD5=[6238b5676e03c48777a4adf876c05dfe]
+link tfmodule to tf--><path codeLine="30" d="M190.6877,52.0251 C172.0867,69.6207 146.3441,93.9718 127.6559,111.6498 " fill="none" id="tfmodule-tf" style="stroke:#312D2A;stroke-width:1.0;"/><!--MD5=[bdac6cf95ae21e0df2990dda9708daf7]
+@startuml
+
+skinparam object {
+    FontColor #312D2A
+    BorderColor #312D2A
+    ArrowColor 312D2A
+    FontName "Oracle Sans"
+    FontSize 15
+}
+
+skinparam objectAttribute {
+    FontColor #312D2A
+    BorderColor #312D2A
+    FontName "menlo"
+    FontSize 13
+}
+
+object "Terraform" as tf #FACD62 {
+    version >= 0.12  && < 0.13
+}
+
+object "OCI Terraform Provider" as tfprovider #94AFAF {
+    version  >= 3.27
+}
+
+object "oci-iam Module" as tfmodule #759C6C {
+    version >= 2.0
+}
+
+tfmodule -d- tfprovider
+tfmodule -d- tf
+@enduml 
+
+PlantUML version 1.2020.23(Sun Dec 13 19:14:58 UTC 2020)
+(GPL source distribution)
+Java Runtime: OpenJDK Runtime Environment
+JVM: OpenJDK 64-Bit Server VM
+Default Encoding: UTF-8
+Language: en
+Country: null
+--></g></svg>
```

Comment:
shouldn't we have a new line here?

Reply:
This is an svg file generated by another tool, intended to be rendered with an image viewer not read with a text editor.
I would prefer to not edit it at all.